src/SourceGenerators/Uno.UI.Tasks/BatchMergeXaml/Utils.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using System.Xml;
+
+namespace Uno.UI.Tasks.BatchMerge
+{
+    class Utils
+    {
+        //The XmlWriter can't handle &#xE0E5 unless we escape/unescape the ampersand
+        public static string UnEscapeAmpersand(string s)
+        {
+            return s.Replace("&amp;", "&");
+        }
+
+        public static string EscapeAmpersand(string s)
+        {
+            return s.Replace("&", "&amp;");
```

Comment:
Codacy found an issue: [Change this call to 's.Replace' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5930374)